### PR TITLE
Let's dockerize the CLI!

### PR DESCRIPTION
**Pull request by quadristan**
We discussed having the CLI in a docker container.

Here are the Dockerfiles.

Disclaimer:
- My laptop is an M1, so I had to create a linux-arm image. I never did multi-arch in docker before, so.. suggestions are welcomed!
- We already identified potential issues with interactive prompts: asking for an email/otp token, and entering a master password may require you to run the container interactively. I wish to not address this in this MR
- I tried to only copy the binary in the Dockerfile... and I got stuck with https://github.com/Dashlane/dashlane-cli/issues/198.  Building the binary and running it from the same image made it work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
         "start": "node dist",
         "pkg:linux": "pkg . -t node18-linux-x64 -o bundle/dcli-linux",
         "pkg:macos": "pkg . -t node18-macos-x64 -o bundle/dcli-macos",
+        "pkg:linux-arm": "pkg . -t node18-linux-arm64 -o bundle/dcli-linux-arm",
```
Comment (quadristan) [inline package.json:27]: I was unsure if I should have added `pkg:linux-arm` to the `pkg` script. Let me know if it is the case

File: Dockerfile.arm [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
```
Comment (quadristan) [inline Dockerfile.arm:1]: Would there be a way to make the arm file depend on the x64 file? 🤔  Let me know, I dislike to just have two things to maintain..

File: package.json (hunk 1 above)
Comment (quadristan) [reply package.json:27]: Also, I did not include the docker build phase in the scripts. Can we think about this in a 2d MR, targeting the github workflows?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
```
Comment (quadristan) [inline Dockerfile:1]: Should I rename this to `Dockerfile.x64` ?

**Comment by Mikescops** [comment]
Please open an issue before opening a PR so we can frame and discuss the work needed. Thanks.

**Pull request closed without merging**